component/file_cache/file_cache.go

```diff
@@ -179,7 +180,11 @@ func (c *FileCache) Configure() error {
 	}
 
 	c.createEmptyFile = conf.CreateEmptyFile
-	c.cacheTimeout = conf.Timeout
+	if config.IsSet(compName + ".timeout-sec") {
```

Comment:
file-cache-timeout is also a cli enable flag so we shall be checking both as you have done in attr-cache

Reply:
actually looks like its not enabled in CLI, I can add it